cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help

```diff
@@ -65,6 +65,9 @@ OPTIONS
        --debug
            Run tests in remote debugging mode, only when the '--skip-tests' flag is not used.
```

Comment:
```suggestion
           Run tests in the remote debugging mode only when the '--skip-tests' flag is not used.
```

Reply:
This too, shall we revisit and verify the best (common) terms to use, and fix in a separate PR maybe?

Reply:
I actually think we should use "remote debug mode", but let's verify and change this properly. @nadundesilva can you create an issue to track these changes please?

Reply:
Created Issue #24275

Reply:
> I actually think we should use "remote debug mode", but let's verify and change this properly. @nadundesilva can you create an issue to track these changes please?

True. +1 for using either of the terms ("remote debugging" or "remote debug"), but for the above context, I also prefer using **remote debug mode**.